### PR TITLE
Create domainiq.py

### PR DESCRIPTION
**Pull request by doomedraven**
thanks @seifreed for point to service ;)

### CODE REVIEW

File: plugins/analytics/public/domainiq.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            }
+    }
+
+    ACTS_ON = ['Hostname', 'Ip', 'Email']
+
+    @staticmethod
+    def process_response(observable, json_result):
+        links = set()
+
+        if isinstance(observable, Ip):
```
Comment (tomchop) [inline plugins/analytics/public/domainiq.py]: Please refactor in methods for each case you're dealing with (Ip, Hostname, Email)

File: plugins/analytics/public/domainiq.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    links.update(
+                        node.active_link_to(observable, 'Registrant', 'DomainIQ'))
+                except ObservableValidationError as e:
+                    logging.error((e, domain_block["registrant"]))
+
+        return links
+
+    @staticmethod
+    def analyze(observable, results):
+        links = set()
```
Comment (tomchop) [inline plugins/analytics/public/domainiq.py]: No use initializing this if you're overwriting the variable in L117

**Comment by tomchop** [comment]
Hey, are you OK to make the requested changes in this PR?

**Comment by doomedraven** [comment]
Will check it tomorrow

**Pull request closed without merging**